src/client-builder.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('ClientBuilder', () => {
       gateway: null,
       gatewayConfigs: {
         Fetch: { config: 'configs' },
-      },
+      } as GatewayConfiguration,
```

Comment:
Leaving this `as GatewayConfiguration` to make tests terser. In reality it needs the full config object, but not for the sake of these specs.